### PR TITLE
add template

### PR DESCRIPTION
**Pull request by dkelsey**

### CODE REVIEW

File: geocoder-data-provider/caddy-minio-pvc-template.yaml [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+      #volume.beta.kubernetes.io/storage-class: gluster-file
+      volume.beta.kubernetes.io/storage-provisioner: netapp.io/trident
+    finalizers:
+    - kubernetes.io/pvc-protection
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}-pv
+  spec:
+    accessModes:
+    - ReadWriteOnce
```
Comment (bolyachevets) [inline geocoder-data-provider/caddy-minio-pvc-template.yaml:346]: seems this need to be ReadWriteMany, as both minio and caddy are trying to access the pvc

File: geocoder-data-provider/caddy-minio-pvc-template.yaml [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+            valueFrom:
+              secretKeyRef:
+                key: minioSecretKey
+                name: ${MINIO_SERVICE_NAME}-${APPLICATION_NAME}
+                #name: minio
+          - name: MINIO_CONFIG_DIR
+            value: /tmp
+          - name: MINIO_DATA_DIR
+            value: /data
+          image: docker-registry.default.svc:5000/openshift/minio
```
Comment (bolyachevets) [inline geocoder-data-provider/caddy-minio-pvc-template.yaml:193]: getting `Failed to pull image "docker-registry.default.svc:5000/openshift/minio": rpc error: code = Unknown desc = error pinging docker registry docker-registry.default.svc:5000: Get "https://docker-registry.default.svc:5000/v2/": dial tcp: lookup docker-registry.default.svc on 142.34.132.200:53: no such host`
replaced this with `image-registry.apps.silver.devops.gov.bc.ca/988040-tools/minio:RELEASE.20201-05-27T22-06-31Z` (that's what barg is using)

Comment (bolyachevets) [inline geocoder-data-provider/caddy-minio-pvc-template.yaml:193]: Also, looks like minio setup runs via `./minio server /data`, so we need to add the following to startup minio pod:
```
 args:
            - server
            - /data
```

**Review by bolyachevets — CHANGES_REQUESTED** [review]
If the suggestions are fine, let me know if you prefer me or yourself to add the changes listed

File: geocoder-data-provider/caddy-minio-pvc-template.yaml (hunk 1 above)
Comment (dkelsey) [reply geocoder-data-provider/caddy-minio-pvc-template.yaml]: i think I had to do that at one point.

File: geocoder-data-provider/caddy-minio-pvc-template.yaml (hunk 2 above)
Comment (dkelsey) [reply geocoder-data-provider/caddy-minio-pvc-template.yaml]: yes.  This also makes sense.  the URL for the image registry has changed.

**Pull request merged**